.github/test-build.yaml

```
@@ -0,0 +1,26 @@
+name: Test Build
+
+on: [push, pull_request]
+
+jobs:
+  build-sphinx:
+    name: Build with Sphinx
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+          cache: "pip"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip setuptools wheel
+          python -m pip install --upgrade -r requirements.txt
+
+      - name: Build docs
+        run: python -b -X dev -W error -m sphinx -n -E -W docs buil
```

There's a reason I like script files over the command line! Could you use the long options here to make it slightly less arcane? The proposed change just fixes a typo.
```suggestion
        run: python -b -X dev -W error -m sphinx -n -E -W docs build
```

---

Oops, that's embarrassing!

I strongly prefer long option names as well, but Sphinx's main entrypoints don't appear to have them (only its legacy, deprecated distutils/setuptools build command). They aren't in [the code](https://github.com/sphinx-doc/sphinx/blob/d82d37073920ce0e2940dccbd25f719cc92a3352/sphinx/cmd/build.py) and nor are they documented anywhere (in `-h`, in the Sphinx docs, or in the `sphinx-build` man page). Its unclear why not, as they do have long option names in [Sphinx's legacy/deprecated setup.py command](https://github.com/sphinx-doc/sphinx/blob/d82d37073920ce0e2940dccbd25f719cc92a3352/sphinx/setup_command.py) but not elsewhere.

I also forgot `--keep-going` and an extra `-b`. `-E` (and `-a`) shouldn't actually be necessary, but I include them included it for explicitness, for users who run the command locally, and in case anything was cached or if we add another Sphinx run on the same runner (linkcheck, etc).